modules/automatic-upgrade-failure.adoc

```diff
@@ -3,22 +3,22 @@
 // * configuration/configure-automatic-upgrades.adoc
 :_mod-docs-content-type: CONCEPT
 [id="automatic-upgrade-failure_{context}"]
-= Automatic upgrade failure
+= Automatic upgrade failure for manifest-installed secured clusters
 
-Sometimes, {product-title} automatic upgrades might fail to install.
+Sometimes, {product-title-short} automatic upgrades might fail to install.
 When an upgrade fails, the status message for the secured cluster changes to `Upgrade failed. Retry upgrade`.
-To view more information about the failure and understand why the upgrade failed, you can check the secured cluster row in the *Clusters* view.
+To view more information about the failure and understand why the upgrade failed, you can check the secured cluster row in the *Clusters* view. For more information, see "Troubleshooting the cluster upgrader".
 
 Some common reasons for the failure are:
 
 * The sensor-upgrader deployment might not have run because of a missing or a non-schedulable image.
 * The pre-flight checks may have failed, either because of insufficient RBAC permissions or because the cluster state is not recognizable.
 This can happen if you have edited {product-title} service configurations or the `auto-upgrade.stackrox.io/component` label is missing.
 * There might be errors in executing the upgrade. If this happens, the upgrade installer automatically attempts to roll back the upgrade.
-+
+
 [NOTE]
 ====
-Sometimes, the rollback can fail as well. For such cases view the cluster logs to identify the issue or contact support.
+Sometimes, the rollback can also fail. For these cases, view the cluster logs to identify the issue or contact support. See "Troubleshooting the cluster upgrader" for more information.
```

Comment:
Nit: What you have isn't wrong, but I would revise your last sentence so that it is consistent with most of the other sentences like this in the docs:
```suggestion
Sometimes, the rollback can also fail. For these cases, view the cluster logs to identify the issue or contact support. For more information, see "Troubleshooting the cluster upgrader".
```